src/main/java/com/bgsoftware/superiorskyblock/commands/player/CmdDisband.java

```diff
@@ -84,7 +84,10 @@ public void execute(SuperiorSkyblockPlugin plugin, SuperiorPlayer superiorPlayer
                         island.getIslandBank().getBalance().multiply(BigDecimal.valueOf(BuiltinModules.BANK.disbandRefund))));
             }
 
-            superiorPlayer.setDisbands(superiorPlayer.getDisbands() - 1);
+            if (superiorPlayer.getDisbands() > 0) {
```

Comment:
Should check if disbands are enabled, and only in this case subtract disband from the player (I know it wasn't there before)